odk2odm/odk_api.py

```diff
@@ -58,6 +69,15 @@ def submissions(base_url, aut, projectId, formId):
     url = f'{base_url}/v1/projects/{projectId}/forms/{formId}/submissions'
     return requests.get(url, auth = aut)
 
+def users(base_url, aut):
+    """Fetch a list of users."""
+    url = f'{base_url}/v1/users'
+    return requests.get(url, auth = aut)
+
+def app_users(base_url, aut, projectId):
+    """Fetch a list of app-users."""
+    url = f'{base_url}/v1/projects/{projectId}/app-users'
+    return requests.get(url, auth = aut)
 
```

Comment:
are 'app-users'  users a subgroup of users, that have access to project {projectId}? Just for my understanding.